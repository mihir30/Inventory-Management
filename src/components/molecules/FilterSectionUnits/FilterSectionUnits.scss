.filter-section {
    display: flex;
    padding: 15px 40px;
}

.date-picker-box-units {
    width: 244px;
    height: 38px;
    padding: 3px 2px;
    color: #475466;
    position: relative;
    margin: 5px 3px 2px 0;
}

.calendar-icon {
    margin: 0px 0px 0px 15px;
}

.month-picker-box {
    width: 130px;
    height: 38px;
    padding: 3px 2px;
    color: #475466;
    position: relative;
    margin: 5px 10px;
}

.condition-picker-box {
    width: 130px;
    height: 38px;
    padding: 3px 2px;
    color: #475466;
    position: relative;
    margin: 5px 10px;
}

.equipment-picker-box {
    width: 216px;
    height: 38px;
    padding: 3px 2px;
    color: #475466;
    position: relative;
    margin: 5px 10px;
}

.btn {
    border: 1px solid #ccd7e6;
    margin: 5px 10px;
    border-radius: 2px;
    color: #475466;
}

.form-select-month {
    border: 1px solid #ccd7e6;
    margin: 5px 10px;
    border-radius: 2px;
    color: #475466;
    height: 35px;
}

.form-select-condition {
    border: 1px solid #ccd7e6;
    margin: 5px 130px;
    border-radius: 2px;
    color: #475466;
    height: 35px;
    width: 130px;
}

.form-select-equipment-units {
    border: 1px solid #ccd7e6;
    margin: 5px 130px;
    border-radius: 2px;
    color: #475466;
    height: 35px;
    width: 216px;
}

.arrow-right {
    margin: 20px 5px 0 0;
    color: #475466;
    &:hover {
        background-color: #f4f5f9;
        border: 0;
        cursor: pointer;
    }
}

.arrow-left {
    margin: 20px 5px 0 0;
    color: #475466;

    &:hover {
        background-color: #f4f5f9;
        border: 0;
        cursor: pointer;
    }
}