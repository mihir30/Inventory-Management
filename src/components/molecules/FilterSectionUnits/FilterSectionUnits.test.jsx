import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import FilterSectionUnits from './FilterSectionUnits';

describe("Filter Section Units Component", () => {
    it('renders the component properly', () => {
        const divEl = document.createElement("div");
        ReactDom.render(<BrowserRouter><FilterSectionUnits></FilterSectionUnits></BrowserRouter>, divEl); 
        
    });


    it('should have the right filter section units component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});