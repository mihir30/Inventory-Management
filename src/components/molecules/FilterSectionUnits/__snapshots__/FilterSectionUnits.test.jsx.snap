// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter Section Units Component should have the right filter section units component snapshot 1`] = `null`;
